src/commands/database-rules-canary.ts

```diff
@@ -0,0 +1,22 @@
+import { Command } from "../command";
+import * as logger from "../logger";
+import * as requireInstance from "../requireInstance";
+import { requirePermissions } from "../requirePermissions";
+import * as metadata from "../database/metadata";
+
+export default new Command("database:rules:canary <ruleset_id>")
+  .description("mark a staged ruleset as the canary ruleset")
+  .option(
+    "--instance <instance>",
+    "use the database <instance>.firebaseio.com (if omitted, uses default database instance)"
+  )
+  .before(requirePermissions, ["firebasedatabase.instances.get"])
+  .before(requireInstance)
+  .action(async (rulesetId: string, options: any) => {
+    const labeled = await metadata.getRulesetLabels(options.instance);
+    await metadata.setRulesetLabels(options.instance, {
+      stable: labeled.stable,
+      canary: rulesetId,
+    });
+    return null;
```

Comment:
If run with `--json` (or, I believe if being run as a script), the return value of the function is what is presented/returned to the user. Should this return something more useful than nothing?

(If it does return nothing, I think specifying `null` is unnecessary)

Reply:
I changed this to return the new label mapping